typing/typecore.ml

```diff
@@ -6859,6 +6918,15 @@ let report_error ~loc env = function
         "@[<2>%s:@ %a@]"
         "This type does not bind all existentials in the constructor"
         (Style.as_inline_code pp_type) (ids, ty)
+  | Bind_existential (reason, id, ty) ->
+      let reason = match reason with
+      | Bind_already_bound -> "already bound existential"
+      | Bind_not_in_scope -> "previously defined existential"
+      | Bind_non_locally_abstract -> "non-locally-abstract type"
+      in
+      Location.errorf ~loc
+        "@[<hov0>This type annotation attempts to bind@ \"%s\"@ %s%s@ \"%a\".@]"
+        (Ident.name id) "to the " reason Printtyp.type_expr ty
```

Comment:
@Octachron changed the way error messages are supposed to refer to inline code (including inline types), see the occurrences of `Style.inline_code (Ident.name id)` and `(Style.as_inline_code Printtype.type_expr) ty` in the rest of the file. You should match that new style instead of using manual quoting here.

Reply:
Ditto